doc/jcli/transaction.md

```diff
@@ -28,3 +28,140 @@ content information of a transaction:
 * `info`
 * `id` to get the **Transaction ID** of the transaction
 * `to-message` to get the hexadecimal encoded message, ready to send with `cli rest message`
+
+
+# Examples
+
+Let's say we have the following utxo
+
+```plaintext
+in_idx: 0
+in_txid: 55762218e5737603e6d27d36c8aacf8fcd16406e820361a8ac65c7dc663f6d1c
+out_addr: ta1sn9u0nxmnfg7af8ycuygx57p5xgzmnmgtaeer9xun7hly6mlgt3pjp0h0rrq782p7z4auve0s4l72egyc88f49mzm8lwuaw7xnw52udky79t3s
+out_value: 100
+```
```

Comment:
This is the output of get REST end point `/utxo`. I would reformulate it to avoid being linked to the output of this command, something like:


| field | value |
|---:|:---|
| UTXO's transaction ID | 55762218e5737603e6d27d36c8aacf8fcd16406e820361a8ac65c7dc663f6d1c |
| UTXO's output index | 0 |
| associated address | ta1sn9u0nxmnfg7af8ycuygx57p5xgzmnmgtaeer9xun7hly6mlgt3pjp0h0rrq782p7z4auve0s4l72egyc88f49mzm8lwuaw7xnw52udky79t3s |
| associated value | 100 |

Also you are using addresses with delegation here which is not relevant to what you are trying to demonstrate. Utilising single addresses(without delegation) would make the code much easier to read.